docs/azure/sdk/authentication/local-development-broker.md

```diff
@@ -0,0 +1,138 @@
+        | WSL         | `ms-appx-web://Microsoft.AAD.BrokerPlugin/{your_client_id}`                                                             |
+        | Linux       | `https://login.microsoftonline.com/common/oauth2/nativeclient`                                                        |
+
+         Replace `{your_client_id}` or `{bundle_id}` with the **Application (client) ID** from the app registration's **Overview** pane.
+
+    1. Select **Configure**.
+
+    To learn more, see [Add a redirect URI to an app registration](/entra/identity-platform/quickstart-register-app#add-a-redirect-uri).
+
+1. Back on the **Authentication** pane, under **Advanced settings**, select **Yes** for **Allow public client flows**.
+1. Select **Save** to apply the changes.
+1. To authorize the application for specific resources, navigate to the resource in question, select **API Permissions**, and enable **Microsoft Graph** and other resources you want to access.
+
+    > [!IMPORTANT]
+    > You must also be the admin of your tenant to grant consent to your application when you sign in for the first time.
+
+### Assign roles
+
+To run your app code successfully with brokered authentication, grant your user account permissions using [Azure role-based access control (RBAC)](/azure/role-based-access-control/overview). Assign an appropriate role to your user account for the relevant Azure service. For example:
+
+- **Azure Blob Storage**: Assign the **Storage Account Data Contributor** role.
+- **Azure Key Vault**: Assign the **Key Vault Secrets Officer** role.
+
+If an app is specified, it must have API permissions set for **user_impersonation Access Azure Storage** (step 6 in the previous section). This API permission allows the app to access Azure storage on behalf of the signed-in user after consent is granted during sign-in.
+
+## Implement the code
+
+:::zone target="docs" pivot="os-windows, os-macos"
+
+.NET and the `Azure.Identity` libraries provide interactive brokered authentication using <xref:Azure.Identity.InteractiveBrowserCredential>. For example, to use `InteractiveBrowserCredential` in a MAUI app to authenticate to Azure Key Vault with the [`SecretClient`](/dotnet/api/azure.security.keyvault.secrets.secretclient), follow these steps:
+
+:::zone-end
+
+:::zone target="docs" pivot="os-linux"
+
+.NET and the `Azure.Identity` libraries provide interactive brokered authentication using <xref:Azure.Identity.InteractiveBrowserCredential>. For example, to use `InteractiveBrowserCredential` in a console app to authenticate to Azure Key Vault with the [`SecretClient`](/dotnet/api/azure.security.keyvault.secrets.secretclient), follow these steps:
+
+:::zone-end
+
+1. Install the [Azure.Identity](https://www.nuget.org/packages/Azure.Identity) and [Azure.Identity.Broker](https://www.nuget.org/packages/Azure.Identity.Broker) packages.
+
+    > [!NOTE]
+    > macOS and Linux support exists in `Azure.Identity.Broker` versions 1.3.0 and later.
```

Comment:
If you manually number the steps in this section, can we have this note appear ONLY for macOS and Linux?

Reply:
Actually, this part is minimal text with minor variations, we could probably just duplicate this part in the zone pivot sections.

Reply:
Go ahead